src/components/common/VerifiedEmail.vue

```diff
@@ -0,0 +1,87 @@
+<template>
```

Comment:
If we like the way this component works, it could easily be moved to bcrs-shared-components repo.

We'd have to pass in a reference to the email verification service (as we do for [NAICS lookup](https://github.com/bcgov/bcrs-shared-components/blob/a341519a9d1f32e8004d168689a5d22c1c22a257/src/components/nature-of-business/NatureOfBusiness.vue#L150)).